tajo-core/src/test/java/org/apache/tajo/engine/parser/TestSQLAnalyzer.java

```diff
@@ -367,6 +364,59 @@ public void testCreateTablePartitionByColumn() throws IOException {
   }
 
   @Test
+  public void testAlterTableAddPartition1() throws IOException {
+    String sql = FileUtil.readTextFile(new File("src/test/resources/queries/default/alter_table_add_partition_1.sql"));
```

Comment:
could you use FileUtil.readTextFileFromResource instead of direct paths? Direct path can cause errors without setting a base directory.